client/src/routes/route.js

```diff
@@ -6,7 +6,7 @@ import CreatingProject from '../pages/CreatingProjectPage';
 import AuthRoute from './AuthRoute';
 import UnauthenticatedRoute from './UnauthenticatedRoute';
 import './routes.scss';
-import AdminSettings from '../pages/SettingsPage/components/AdminSettingsPage/AdminSettings';
+import Settings from '../pages/SettingsPage/components/AdminSettingsPage/Settings';
```

Comment:
'../pages/SettingsPage/components/AdminSettingsPage/Settings' file doesn't exist
Route should be corrected